doc/make.jl

```diff
@@ -0,0 +1,144 @@
+# Install dependencies needed to build the documentation.
+ENV["JULIA_PKGDIR"] = joinpath(@__DIR__, "deps")
+Pkg.init()
+write(Pkg.dir("REQUIRE"), "Documenter")
```

Comment:
Sure, sounds good to me. Would you suggest using `Pkg.pin` to do that or specifying the versions in  the `REQUIRE` file directly?